sim-lib/src/lib.rs

```diff
@@ -213,25 +236,36 @@ pub struct Simulation {
     total_time: Option<time::Duration>,
     /// The number of activity results to batch before printing in CSV.
     print_batch_size: u32,
+    /// The expected payment size for the network.
+    expected_payment_msat: u64,
+    /// The number of times that the network sends its total capacity in a month of operation when generating random
+    /// activity.
+    activity_multiplier: f64,
 }
 
 const DEFAULT_PRINT_BATCH_SIZE: u32 = 500;
 
 impl Simulation {
     pub fn new(
         nodes: HashMap<PublicKey, Arc<Mutex<dyn LightningNode + Send>>>,
-        activity: Vec<ActivityDefinition>,
+        activity: Option<Vec<ActivityDefinition>>,
         total_time: Option<u32>,
         print_batch_size: Option<u32>,
     ) -> Self {
         let (shutdown_trigger, shutdown_listener) = triggered::trigger();
         Self {
             nodes,
-            activity,
+            activity: match activity {
+                Some(a) => a,
+                None => Vec::new(),
+            },
             shutdown_trigger,
             shutdown_listener,
             total_time: total_time.map(|x| Duration::from_secs(x as u64)),
             print_batch_size: print_batch_size.unwrap_or(DEFAULT_PRINT_BATCH_SIZE),
+            // Approx $10 at the time of writing.
+            expected_payment_msat: 3_800_000,
+            activity_multiplier: 2.0,
```

Comment:
Should we make these constants so they are easier to find? Especially to code readers/reviewers.

If at some point we want to make this configurable, then we can simply default to them when parsing the config.